### PR TITLE
[fix #38] Euen.38.rebar3

### LINKED ISSUE #38: Move from erlang.mk to rebar3
Migrate to rebar3 as described [here](https://github.com/inaka/guidelines/blob/master/ERLANG.md#project-setup).

This is requested by esl/TODO#62

### PR DESCRIPTION
**Pull request by Euen**
We must consider this warning

> ===> WARNING: It is recommended to use {branch, Name}, {tag, Tag} or {ref, Ref}, otherwise updating the dep may not work as expected.

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -1,19 +1,94 @@
...
+    {plt_location, local},
+    {base_plt_apps, [stdlib, kernel]},
+    {base_plt_location, global}
+]}.
+
+%% == Shell ==
+
+{shell, [{apps, [sync]}]}.
+
+%% == xref ==
```
Comment (elbrujohalcon) [inline rebar.config]: We don't usually use `xref` from command line, I think you can remove the xref-related lines from `rebar.config`.
We run xref on the `zipper_meta_SUITE` already :)

File: rebar.config [hunk 2]
```diff
@@ -1,19 +1,94 @@
...
+    {plt_apps, top_level_deps},
+    {plt_extra_apps, []},
+    {plt_location, local},
+    {base_plt_apps, [stdlib, kernel]},
+    {base_plt_location, global}
+]}.
+
+%% == Shell ==
+
+{shell, [{apps, [sync]}]}.
```
Comment (elbrujohalcon) [inline rebar.config]: `sync` is not in any `deps` list. I'm pretty sure `rebar3 shell` doesn't work.

Comment (Euen) [inline rebar.config]: It was added as a profile above.

File: src/zipper.erl [hunk 3]
```diff
@@ -50,6 +50,9 @@
                   , is_modified => boolean()
                   }.
 
+
```
Comment (elbrujohalcon) [inline src/zipper.erl]: @Euen please remove these empty lines

File: src/zipper.app.src [hunk 4]
```diff
@@ -4,7 +4,7 @@
    {description, "Generic Zipper Implementation for Erlang"},
    {vsn, "0.2.0"},
    {applications, [kernel, stdlib]},
-   {modules, []},
+   {modules, [zipper, zipper_default]},
```
Comment (elbrujohalcon) [inline src/zipper.app.src]: This change is not needed.

**Pull request merged**